runtime-modules/content/src/lib.rs

```diff
@@ -1851,6 +1851,12 @@ decl_module! {
             // Ensure top bid exists
             let top_bid = english_auction.ensure_top_bid_exists::<T>()?;
 
+            // ensure claiming member is top auction bidder
+            ensure!(
+                top_bid.bidder_id == member_id,
+                Error::<T>::MemberClaimingIsNotWinner
+            );
```

Comment:
I don't think that's actually the behaviour we want, this would impact already designed and implemented user experience. The auction should still be settle-able by anyone (or we can restrict it to winner&owner if needed). Instead, I think it's a matter of passing `top_bid.bidder_id` as the auction winner to `Self::complete_auction` instead of `member_id`.

Reply:
Ah completely missed Bedeho's comment, that's the same thing